var/spack/repos/builtin/packages/gh/package.py

```diff
@@ -16,6 +16,14 @@ class Gh(GoPackage):
 
     license("MIT")
 
+    version("2.63.2", sha256="2578a8b1f00cb292a8094793515743f2a86e02b8d0b18d6b95959ddbeebd6b8d")
+    version("2.63.1", sha256="b9a90118dfb46204dbcc0d09c2073d48f35b6f640b4db33fbaa24892fed56c8d")
+    version("2.63.0", sha256="c5309db9707c9e64ebe264e1e2d0f893ecead9056d680b39a565aaa5513d2947")
+    version("2.62.0", sha256="8b0d44a7fccd0c768d5ef7c3fbd274851b5752084e47761f146852de6539193e")
+    version("2.61.0", sha256="bf134281db2b65827426e3ad186de55cb04d0f92051ca2e6bd8a7d47aabe5b18")
+    version("2.60.1", sha256="9e9337c2564894c4cd32b2ac419611263c3e870e95567811365aacd4be5dd51d")
+    version("2.60.0", sha256="1936a80a668caef437b2f409eaa10e48613a3502db7da9eea011b163769218a7")
+    version("2.59.0", sha256="d24ed01e5aa1e8f42b397333462f9cd5c54e4845a5142044381fc8eb713fa001")
```

Comment:
@albestro we're missing a minimum version constraint on go (from `go.mod` in the repo) can you update that?

```
depends_on("go@1.22.5:", when="@2.56.0:")
```

Reply:
good catch! thanks for noticing it 😉